packages/pigeon/platform_tests/windows_unit_tests/README.md

```diff
@@ -0,0 +1,15 @@
+# windows_unit_tests
+
+A new flutter plugin project.
```

Comment:
Probably should update this description.  A note with how to execute the tests would be helpful too.